tripy/tripy/frontend/trace/trace.py

```diff
@@ -32,25 +34,25 @@ class Trace:
 
     def __init__(
         self,
-        tensors: Sequence["tripy.Tensor"],
-        inputs: Sequence["tripy.Tensor"] = [],
+        tensors: Sequence["tripy.frontend.trace.tensor.TraceTensor"],
+        inputs: Sequence["tripy.frontend.trace.tensor.TraceTensor"] = [],
```

Comment:
We import `TraceTensor` here anyway:
```suggestion
        tensors: Sequence[TraceTensor],
        inputs: Sequence[TraceTensor] = [],
```